index.rst

```diff
@@ -48,10 +48,63 @@
 
    **This technote is not yet published.**
 
-   This proposal outlines how observing tasks will be flowed from the planning stage through execution, then linked to data analysis technotes and/or notebooks.
+   This proposal outlines how observing tasks will be flowed from the planning stage through execution,
+   then linked to data analysis technotes and/or notebooks.
+
+Motivation for this proposal
+============================
+
+    * We want central repository for all information related to a task executed at night.
+    * We want to have a way to know how and when the tasks were run and how the data was processed.
+    * Search and find tasks and their results.
```

Comment:
Night plans need to point to tasks with procedures rather than having the task description in the plan

Reply:
What do think? Would it be interesting to have some sort of simple engineering task proposal platform? It'd be to mainly make sure all the information is contained and properly described when creating night notebooks or tasks.

Reply:
I think it needs to be included who creates the tickets?